cmd/template/cluster/flag.go

```diff
@@ -91,6 +94,13 @@ func (f *flag) Validate() error {
 			return microerror.Maskf(invalidFlagError, "--%s must only contain alphanumeric characters, and start with a letter", flagClusterID)
 		}
 
+		if f.ControlPlaneSubnet != "" {
+			matchedSubnet, err := regexp.MatchString("^24|25|26|27$", f.ControlPlaneSubnet)
+			if err == nil && !matchedSubnet {
```

Comment:
Can you please point me to which values would be correct

Reply:
I would say correct values are /20-/25.

26 and 27 not really usable due to subneting